sdk/aqueduct/table_artifact.py

```diff
@@ -156,6 +172,93 @@ def save(self, config: SaveConfig) -> None:
+                Optional severity associated with the check created with this expectations
+        Returns:
+            A check artifact that represent the validation result of running the expectation provided on the table.
+        """
+
+        def great_expectations_check_method(table: pd.DataFrame) -> bool:
+            data_context_config = DataContextConfig(
+                datasources={
+                    "my_pandas_datasource": DatasourceConfig(
+                        class_name="PandasDatasource",
+                    )
+                },
+                store_backend_defaults=FilesystemStoreBackendDefaults(root_directory="/tmp"),
+            )
+
+            context = BaseDataContext(project_config=data_context_config)
+
+            # create and load Expectation Suite
+            expectation_suite_name = "aq_check_expectation"
+            context.create_expectation_suite(
+                expectation_suite_name=expectation_suite_name, overwrite_existing=True
+            )
+            suite = context.get_expectation_suite(expectation_suite_name=expectation_suite_name)
+            expectation_config = ExpectationConfiguration(expectation_name, expectation_args)
+            suite.add_expectation(expectation_config)
+
+            # We create a custom datasource that will allow us to load a in-memory dataframe to validate the expectation quite
+            datasource_yaml = f"""
+            name: my_pandas_datasource
+            class_name: Datasource
+            module_name: great_expectations.datasource
+            execution_engine:
+                module_name: great_expectations.execution_engine
+                class_name: PandasExecutionEngine
+            data_connectors:
+                aq_runtime_dataconnecter:
+                    class_name: RuntimeDataConnector
+                    batch_identifiers:
+                        - default_identifier_name
+            """
+            context.add_datasource(**yaml.load(datasource_yaml, Loader=ruamel.yaml.Loader))
+
+            df: pd.DataFrame = table
+            runtime_batch_request = RuntimeBatchRequest(
+                datasource_name="my_pandas_datasource",
+                data_connector_name="aq_runtime_dataconnecter",
+                data_asset_name="aq_table_check",
+                runtime_parameters={"batch_data": df},
+                batch_identifiers={"default_identifier_name": "aq_table_check"},
+            )
+
+            # Constructing Validator by passing in RuntimeBatchRequest
+            my_validator: Validator = context.get_validator(
+                batch_request=runtime_batch_request,
+                expectation_suite=suite,
+            )
+
+            # run validation to return the result
+            result = my_validator.validate()
+            return bool(result.success)
+
+        return self._apply_check_to_table(
+            great_expectations_check_method,
+            "ge_table_check: {%s}" % expectation_name,
```

Comment:
"ge" might be slightly confused with "greater than or equal to", should we just call it "great expectations check: {name}"?